README.md

```diff
@@ -68,6 +68,13 @@ version for the underlying OS and
 This image type falls under the customer [SLA](https://github.com/customer-terms/github-online-services-sla) for actions. GA images are eventually deprecated according to our guidelines as we only support the
 latest 2 versions of an OS.
 
+#### Migration Process

```

Comment:
Can we call this heading "Latest" as there are three phases to the image lifecycle. We describe the migration process in the text below.